src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java

```diff
@@ -61,4 +68,20 @@ public void iAssignARankToAUserWithUsername(String rank, String username) throws
                         .with(AuthenticationStepDefs.authenticate()))
                 .andDo(print());
     }
+
+    @And("It has been assigned the rank {string} to a user with username {string}")
+    public void itHasBeenAssignedTheRankToAUserWithUsername(String rank, String username) throws Exception {
+        stepDefs.result = stepDefs.mockMvc.perform(
+                get("/committeeMembers/{id}", 1L)
```

Comment:
The committeeMember identifier is here fixed to 1. Better to keep dynamic, get the id using findBy or storing the URI of the new resource retrieving the Location header from the POST response when it is created